packages/workspace/src/utils/app-root.ts

```diff
@@ -1,8 +1,11 @@
 import { fileExists } from './fileutils';
 import * as path from 'path';
 
-// TODO: vsavkin normalize the path
-export const appRootPath = pathInner(__dirname);
+function normalizeBackslashes(dir) {
```

Comment:
While looking at this, I think we should stay path delimiter agnostic instead of forcing windows into using forward slashes. Otherwise we could run into other issues in unknown places where `app-root` is being used. Apart from this it's a good start for a discussion. I'm excited to see the CI/CD results if this fixes it or if implications of this change are correctly revealed by the CI/CD. 

Let's wait for some maintainer comments. Thank you for creafting this PR 👍

Reference:
https://nodejs.org/api/path.html#path_path_delimiter

Reply:
I guess you meant https://nodejs.org/api/path.html#path_path_sep not delimiter. You may be right, let's wait for maintainers feedback.

Reply:
Oh yes, I just searched for a path example and cited the wrong section. I wanted to mention the path separator of course. Thanks for pointing out 👍